src/main/features/response/views/eligibility/send-your-response-by-email.njk

```diff
@@ -16,9 +16,84 @@
         </div>
       </div>
 
-
       <h2 class="heading-medium">{{ t('Follow these steps') }}</h2>
 
+      <ul class="list list-number">
+        <li>
+          {% if draft.response.type === ResponseType.FULL_ADMISSION %}
+            {{ t('Download the <span class="bold-small">admission form.</span>') | safe }}
```

Comment:
are these supposed to be linked?

Reply:
links are now at the bottom of the page

Reply:
for a visual: 
https://hmcts-cmc-prototype.herokuapp.com/defendant/full/d-5/task-list/do-you-owe-money/forms?handoffs=4